TestSuite/repo.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, tmpdir: Path):
     def __del__(self):
         shutil.rmtree(self.path, ignore_errors=True)
 
-    def setup_one_pack(self, name) -> Pack:
+    def setup_one_pack(self, name, marketplace='xsoar') -> Pack:
```

Comment:
This is a bit problematic as the pack can be suitable for xsoar, xsiam or both. Let's set `marketplace` as a list. If it will be None then set it to have xsoar in it like this:
```suggestion
    def setup_one_pack(self, name, marketplace=None) -> Pack:
        if not marketplace:
            marketplace = ['xsoar']
```

Don't forget to add it to dicstring!

Reply:
done